Tests/Tests/CDTQIndexTests.m

```diff
@@ -0,0 +1,174 @@
+//
+//  CDTQIndexTests.m
+//  CDTQIndexTests
+//
+//  Created by Al Finkelstein on 04/21/2015.
+//  Copyright (c) 2015 Cloudant. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+
+#import <CloudantSync.h>
+#import <CDTQIndex.h>
+#import "Matchers/CDTQContainsAllElementsMatcher.h"
+
+SpecBegin(CDTQIndex)
+
+describe(@"When creating an instance of index", ^{
+
+    __block NSArray *fieldNames;
+    __block NSString *indexName;
+    
+    beforeAll(^{
+        fieldNames = @[ @"name", @"age" ];
+        indexName = @"basic";
+    });
+    
+    it(@"constructs an index instance with the default index type", ^{
+        CDTQIndex *index = [CDTQIndex index:indexName withFields:fieldNames];
+        
+        expect(index.indexName).to.equal(@"basic");
+        expect(index.fieldNames).to.containsAllElements(@[ @"name", @"age" ]);
+        expect(index.indexType).to.equal(@"json");
+        expect(index.indexSettings).to.beNil;
```

Comment:
I think your `beNil` calls require parentheses to work: https://github.com/specta/expecta/issues/50